views/index.erb

```diff
@@ -0,0 +1,36 @@
+<html>
+<head>
+<title>Lauren Granger's Awesome Website of Awesomeness</title>
+</head>
+
+<header>
+  <h1>Welcome to Lauren's Awesome Website of Awesomeness</h1>
+<nav>
+  <ul>
+    <strong><li><a href="index">Home</a></li></strong>
+    <li><a href="about_me">All About Lauren</a></li>
+    <li><a href="blog">Blog</a></li>
+    <li><a href="projects">Projects</a></li>
+  </ul
+</nav>
+<img src="./images/break_line.png"/>
+</header>
+
+<body>
+  <p>
+    <img src = "./images/silly_penny.png" alt = "Silly Penny"/>
+  </p>
+</body>
+
+<footer>
+  <img src="./images/break_line.png"/>
+    <p>
```

Comment:
Since the `img` tag is self-closing, you don't need to indent the subsequent `p`